wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Very nice addition, i really like this script 😛

Reply:
:D